src/test/scala/org/camunda/feel/examples/SpecExampleTest.scala

```diff
@@ -75,6 +75,10 @@ class SpecExampleTest extends AnyFlatSpec with Matchers with FeelIntegrationTest
 
   }
 
+  it should "compute long list" in {
+    evalWithContext(""" for x in 0..1000000 return "Hi there " + string(x) """)
```

Comment:
:wrench: We should verify the result and move the test into a different class. 

We have some test cases with big lists in `InterpreterListExpressionTest`.